com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -114,6 +151,9 @@ public UnityRLInitParameters Initialize(CommunicatorInitParameters initParameter
 
                     throw new UnityAgentsException("ICommunicator.Initialize() failed.");
                 }
+
+                // Even if we initialize, we still want to check to make sure that we inform users of minor version
```

Comment:
This comment feels out of place - should it be in CheckCommunicationVersionsAreCompatible?

Reply:
Yeah, forgot to move it.

Reply:
actually, I wanted to do the check outside of the initalization if statement.  Thanks for catching that.